gprproject/gprbuild.py

```diff
@@ -194,6 +201,10 @@ def run(self, args: list[str], **kwargs) -> int:
         for key, value in self.variables.items():
             cmd.append(f"-X{key}={value}")
 
+        if self.gpr_opts is not None:
```

Comment:
This block of code can be moved into the `if cmd_name == "gprbuild":` block above. Otherwise, arguments may be incorrectly added to gprinstall, which supports fewer arguments than gprbuild. Based on the parameter description, I assumed that `gpr_opts` is intended to be used exclusively for gprbuild.

Reply:
The `gpr_opts` option was already in the code base but not implemented. It is not specific to gprbuild, but should be available to gprbuild, gprclean and gprinstall.